packages/shared/src/graphql/feedSettings.ts

```diff
@@ -72,14 +65,14 @@ export const SEARCH_TAGS_QUERY = gql`
 `;
 
 export const FEED_SETTINGS_QUERY = gql`
-  query TagCategories($loggedIn: Boolean!) {
+  query TagCategories {
```

Comment:
While we're at it, could we maybe rename this to something else?
Like FeedSettings? I feel like the name `TagCategories` is a bit misleading, as it also fetches `feedSettings` and `advancedSettings` in addition the the actual `tagsCategories`?

Maybe call it FeedSettings or something.
```suggestion
  query FeedSettings {
```

I was thinking about sneaking it into one of my YouTube changes, but since you're already here. Thoughts @rebelchris?

Reply:
Yeah, sure, maybe `FeedConfig` can work? Just a bit odd when calling it since it has a property called the exact same thing (feedSettings). Wdyt?

Reply:
Didn't really like FeedSettings tbh, so
FeedPreferences is a much better name I think.

Reply:
Works for me :)